crates/bitwarden/src/client/auth_settings.rs

```diff
@@ -31,7 +32,13 @@ pub enum Kdf {
 }
 
 impl AuthSettings {
+    #[cfg(feature = "internal")]
     pub fn new(response: PreloginResponseModel, email: String) -> Self {
+        use crate::util::{
+            default_argon2_iterations, default_argon2_memory, default_argon2_parallelism,
+            default_pbkdf2_iterations,
+        };
```

Comment:
What is the benefit of moving use into functions? Just take advantage of the existing `feature` check?

Reply:
Yeah, pretty much. Otherwise it's another cfg that we need to add.